Thực Đơn/ViewController.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+final class ViewController: UIViewController {
+    
+    @IBOutlet private  weak var containerViewHome: UIView!
+    @IBOutlet private weak var containerViewFavorite: UIView!
+    @IBOutlet private  weak var containerViewDiet: UIView!
+    @IBOutlet private weak var containerViewIngredient: UIView!
+    
+    @IBOutlet private weak var TabBarCollection: UICollectionView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTabBar()
+        defaultView()
+    }
+    
+    func defaultView() {
+        [
+            containerViewFavorite,
+            containerViewDiet,
+            containerViewIngredient
+        ].forEach { $0?.isHidden = true }
+    }
+    
+    func configureTabBar() {
+        TabBarCollection.selectItem(at: IndexPath(item: 0, section: 0), animated: true, scrollPosition: .centeredHorizontally)
+        TabBarCollection.register( TabBarCell.self, forCellWithReuseIdentifier: Instance.idTabBarCell )
+        TabBarCollection.collectionViewLayout.invalidateLayout()
+    }
+}
+
+extension ViewController : UICollectionViewDelegate,
+                           UICollectionViewDataSource,
+                           UICollectionViewDelegateFlowLayout {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return Instance.tabBars.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return CGSize(width: (self.view.frame.width - 24) / 4, height: 100)
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let item = Instance.tabBars[indexPath.row]
+        let cell = TabBarCollection.dequeueReusableCell(withReuseIdentifier: Instance.idTabBarCell, for: indexPath) as? TabBarCell
+        guard let myCell = cell else { fatalError() }
```

Comment:
```suggestion
        guard let myCell = cell else { return UICollectionViewCell() }
```
faltalError sẽ làm Crash App